components/Graphs/MetricCard.tsx

```diff
@@ -0,0 +1,71 @@
+import EChartsReact from "echarts-for-react";
+import { FaArrowUp, FaEllipsisVertical } from "react-icons/fa6";
+import Card from "components/atoms/Card/card";
+import Button from "components/shared/Button/button";
+import { StatsType } from "lib/hooks/api/useFetchMetricStats";
+
+type MetricCardProps = {
+  stats: StatsType[] | undefined;
+  variant: "stars" | "forks";
+};
+
+export default function MetricCard({ stats, variant }: MetricCardProps) {
+  const seriesData = stats?.map((stat) => (variant === "stars" ? stat.star_count : stat.forks_count));
+  const bucketData = stats?.map((stat) => new Date(stat.bucket).toDateString());
+
+  const option = {
+    xAxis: {
+      type: "category",
+      data: bucketData,
+      show: false,
+    },
+    yAxis: {
+      type: "value",
+      show: false,
+    },
+    tooltip: {
+      trigger: "axis",
+      axisPointer: {
+        type: "shadow",
+      },
+    },
+    series: [
+      {
+        data: seriesData,
+        symbol: "none",
+        type: variant === "stars" ? "line" : "bar",
+      },
+    ],
+    color: "hsla(19, 100%, 50%, 1)",
+  };
+
+  const total = seriesData?.reduce((stat, currentValue) => (stat || 0) + (currentValue || 0), 0);
+
+  return (
+    <Card className="w-full max-w-sm h-fit p-5 pl-6">
+      <section className="flex justify-between items-center">
+        <p className="text-xl font-semibold capitalize">{variant} per day</p>
+        <Button variant="default" className="!p-1 rounded-full">
+          <FaEllipsisVertical />
+        </Button>
+      </section>
+
+      <section className="flex justify-between items-center">
+        <div className="basis-1/2 w-full">
+          <p className="text-5xl font-bold">{total}</p>
+        </div>
+        <div className="h-fit w-full max-w-[12rem]">
+          <EChartsReact option={option} style={{ height: "100%", width: "100%" }} />
+        </div>
+      </section>
+
+      <section className="flex items-center gap-2">
+        <div className="flex items-center gap-1 px-2 py-0.5 border-green-300 border-2 rounded-full bg-green-200">
+          <FaArrowUp className="text-green-500" />
+          <p className="text-green-700">10%</p>
```

Comment:
The green on green causes colour contrast failures. We need to darken the text.

![CleanShot 2024-03-26 at 16 54 57](https://github.com/open-sauced/app/assets/833231/9de55bc7-c0af-4a9e-98d7-1111e0d58941)

Setting it to `text-green-800` seems to do the trick.

![CleanShot 2024-03-26 at 16 58 03](https://github.com/open-sauced/app/assets/833231/ab2bc14d-309e-4fcb-98e6-e6702ffcb1e1)

```suggestion
          <p className="text-green-800">10%</p>
```

Reply:
good call! Actually instead of making the text darker, let's make the background lighter 

bg-green-100 seems to work just fine